### PR TITLE
Search document

### PR DESCRIPTION
**Pull request by nhannn-2212**

### CODE REVIEW

**Comment by nhannn-2212** [comment]
image

**Comment by nhannn-2212** [comment]
Description: Search document
Ticket: https://edu-redmine.sun-asterisk.vn/issues/24420
image

**Comment by nhannn-2212** [comment]
ready

File: app/views/layouts/_header.html.erb [hunk 1]
```diff
@@ -1,28 +1,34 @@
...
+                </li>
+              </ul>
+            </li>
+          <% else %>
+            <li><%= link_to t("login"), login_path %></li>
+          <% end %>
+        </ul>
+        <%= form_tag("/search", method: :get, remote: true, class: "form-inline my-2 my-lg-0") do %>
+          <%= text_field_tag :search, params[:search], class: "form-control mr-sm-2" %>
+          <%= submit_tag t("search"), :name => nil, class: "btn btn-outline-success my-2 my-sm-0" %>
```
Comment (vanvtt-0952) [inline app/views/layouts/_header.html.erb:29]: `:name => nil`, phải cần ri ah chú, xóa dc ko

Comment (nhannn-2212) [reply app/views/layouts/_header.html.erb:29]: dạ đc chị :D

**Comment by nhannn-2212** [comment]
ready

**Pull request merged**